concepticondata/conceptlists/Gerardi-2020-244.tsv

```diff
@@ -0,0 +1,245 @@
+Gerardi-2020-244-63	63	FIREFLY	206	FIREFLY	VAGALUME / PIRILAMPO
+Gerardi-2020-244-64	64	FIREWOOD	10	FIREWOOD	LANHA
+Gerardi-2020-244-65	65	FISH	227	FISH	PEIXE
+Gerardi-2020-244-66	66	FLAT	1633	FLAT	PLANO / LISO
+Gerardi-2020-244-67	67	FLESH	2615	FLESH OR MEAT	CARNE
+Gerardi-2020-244-68	68	FLOUR	1594	FLOUR	FARINHA
+Gerardi-2020-244-69	69	FLOWER	239	FLOWER	FLOR
+Gerardi-2020-244-70	70	FLY	1441	FLY (MOVE THROUGH AIR)	VOAR
+Gerardi-2020-244-71	71	FLY(n)	1504	FLY (INSECT)	MOSCA
+Gerardi-2020-244-72	72	FOOT	1301	FOOT	PÉ
+Gerardi-2020-244-73	73	FOUR	1500	FOUR	QUATRO
+Gerardi-2020-244-74	74	FROG	503	FROG	SAPO
+Gerardi-2020-244-75	75	FRUIT	1507	FRUIT	FRUTA
+Gerardi-2020-244-76	76	FULL	1429	FULL	CHEIO
+Gerardi-2020-244-77	77	GATHER	7	GATHER	(A)JUNTAR / RECOLHER
+Gerardi-2020-244-78	78	GIRL	1646	GIRL	MENINA
+Gerardi-2020-244-79	79	GIVE	1447	GIVE	DAR
+Gerardi-2020-244-80	80	GO	695	GO	IR
+Gerardi-2020-244-81	81	GO_UP	705	GO UP (ASCEND)	SUBIR
+Gerardi-2020-244-82	82	GOOD	1035	GOOD	BOM / BEM
+Gerardi-2020-244-83	83	GRANDFATHER	1383	GRANDFATHER	AVÔ
+Gerardi-2020-244-84	84	GRASS	606	GRASS	GRAMA
+Gerardi-2020-244-85	85	GREEN	2382	BLUE OR GREEN	VERDE / AZUL
+Gerardi-2020-244-86	86	GROW	479	GROW	CRESCER
+Gerardi-2020-244-87	87	HAND	1277	HAND	MÃO
+Gerardi-2020-244-88	88	HAIR	2648	HAIR (HEAD)	CABELO
+Gerardi-2020-244-89	89	HAMMOCK	2019	HAMMOCK	REDE
+Gerardi-2020-244-90	90	HAPPY	1495	HAPPY	FELIZ / CONTENTE
+Gerardi-2020-244-91	91	HAWK	307	HAWK	GAVIÃO
+Gerardi-2020-244-92	92	HEAL	499	CURE	CURAR / SARAR
+Gerardi-2020-244-93	93	HEAR/LISTEN			ESCUTAR / OUVIR
+Gerardi-2020-244-94	94	HEART	1223	HEART	CORAÇÃO
+Gerardi-2020-244-95	95	HEAVY	1210	HEAVY	PESADO
+Gerardi-2020-244-96	96	HEDGEHOG	1797	HEDGEHOG	OURIÇO / PORCO-ESPINHO
+Gerardi-2020-244-97	97	HELP	1897	HELP	AJUDAR
+Gerardi-2020-244-98	98	HEN	1514	HEN	GALINHA
+Gerardi-2020-244-99	99	HIDE	2486	HIDE	ESCONDER
+Gerardi-2020-244-100	100	HIT	1433	HIT	BATER
+Gerardi-2020-244-101	101	HOLE	1667	HOLE	BURACO
+Gerardi-2020-244-102	102	HOT	1286	HOT	QUENTE
+Gerardi-2020-244-103	103	HOUSE	1252	HOUSE	CASA
+Gerardi-2020-244-104	104	HOWLER_MONKEY	433	HOWLER MONKEY	(MACACO) BUGIO / GUARIBA
+Gerardi-2020-244-105	105	HUSBAND	1200	HUSBAND	MARIDO
+Gerardi-2020-244-106	106	JACARÉ/CAIMAN/CROCODILE	1857	CROCODILE	JACARÉ
+Gerardi-2020-244-107	107	JAGUAR	1250	JAGUAR	ONÇA
+Gerardi-2020-244-108	108	KILL	1417	KILL	MATAR
+Gerardi-2020-244-109	109	KNEE	1371	KNEE	JOELHO
+Gerardi-2020-244-110	110	KNIFE	1352	KNIFE	FACA
+Gerardi-2020-244-111	111	KNOW	3626	KNOW	SABER
+Gerardi-2020-244-112	112	LAKE	624	LAKE	LAGO / LAGOA
+Gerardi-2020-244-113	113	LAUGH	1355	LAUGH	RIR
+Gerardi-2020-244-114	114	LEAF	628	LEAF	FOLHA
+Gerardi-2020-244-115	115	LEARN	504	LEARN	APRENDER
+Gerardi-2020-244-116	116	LEG	1297	LEG	PERNA
+Gerardi-2020-244-117	117	LAY	2071	LAY (VERB)	DEITAR(-SE)
+Gerardi-2020-244-118	118	LIVER	1224	LIVER	FÍGADO
+Gerardi-2020-244-119	119	LONG	2143	LONG OR TALL	COMPRIDO / LONGO / ALTO
+Gerardi-2020-244-120	120	LOOK_AT	1819	LOOK	OLHAR
```

Comment:
"Look at" seems more specific than just LOOK so I would unmap it.

Reply:
Author's decision as well, but I agree with @CarolinHu here. The comparanda suggests a general LOOK. I will change it, but we can always revert.

Reply:
I would agree with @CarolinHu here, and the comparanda seems to support it. I will keep it as it is for now, waiting for author's comments (it is his original mapping).